lib/duo.go

```diff
@@ -242,7 +242,11 @@ func (d *DuoClient) DoStatus(txid, sid string) (auth string, err error) {
 	err = json.NewDecoder(res.Body).Decode(&status)
 
 	if status.Response.Result == "SUCCESS" {
-		auth, err = d.DoRedirect(status.Response.ResultURL, sid)
+		if status.Response.ResultURL != "" {
+			auth, err = d.DoRedirect(status.Response.ResultURL, sid)
+		} else if status.Response.Cookie != ""  {
+			auth = status.Response.Cookie
+		}
```

Comment:
What about the case where `status.Response.ResultURL == "" && status.Response.Cookie == ""`?

Reply:
this is how it was done previously: https://github.com/segmentio/aws-okta/pull/68/files#diff-2461124d964c26f736e12a1edae3f065L243 . So im okay with this

Reply:
you're right. `else` makes more sense then `else if` here

Reply:
I'm going to approve since this is clearly better than before, but this case needs covering IMO

Reply:
@eddieramirez can you make this an `else` instead? once we have that I will merge

Reply:
done

Reply:
Thanks. Sorry to be a PITA